src/lib.rs

```diff
@@ -46,6 +46,10 @@ pub mod services {
         pub use api_server::{Api, ApiServer as Server};
     }
 
+    pub mod packet_verifier {
+        include!(concat!(env!("OUT_DIR"), "/helium.packet_verifier.rs"));
```

Comment:
Follow the convention of the other services and expose `Client`, `Server` and the interface that needs implementing

Reply:
There is no client and server, it's just a daemon. If that's the case, should I just put this stuff somewhere else?